CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [3.2.0](https://github.com/serverless/serverless/compare/v3.1.0...v3.2.0) (2022-02-02)
+
+### Features
+
+- **CLI Onboarding:** Improve deploy messaging ([#10623](https://github.com/serverless/serverless/pull/10623)) ([97fda34](https://github.com/serverless/serverless/commit/97fda34b29ecd3711bcd4b94c741ff9b17663188)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **CLI:** Expose installation type by `serverless-tencent` version info ([#10621](https://github.com/serverless/serverless/pull/10621)) ([4c5f834](https://github.com/serverless/serverless/commit/4c5f8349050cb870f80f210be4f93f7fb55bd71c)) ([Mariusz Nowak](https://github.com/medikoo))
```

Comment:
I think it was merged as _fix_, but if not it should also be considered a _maintainance_ patch